Source/SwiftLintBuiltInRules/Rules/Idiomatic/UnneededBreakInSwitchRule.swift

```diff
@@ -65,3 +82,48 @@ private final class UnneededBreakInSwitchRuleVisitor: ViolationsSyntaxVisitor {
         violations.append(statement.item.positionAfterSkippingLeadingTrivia)
     }
 }
+
+private final class UnneededBreakInSwitchRewriter: SyntaxRewriter, ViolationsSyntaxRewriter {
+    private(set) var correctionPositions: [SwiftSyntax.AbsolutePosition] = []
+    let locationConverter: SourceLocationConverter
+    let disabledRegions: [SourceRange]
+
+    init(locationConverter: SourceLocationConverter, disabledRegions: [SourceRange]) {
+        self.locationConverter = locationConverter
+        self.disabledRegions = disabledRegions
+    }
+
+    override func visit(_ node: SwitchCaseSyntax) -> SwitchCaseSyntax {
+        guard node.statements.count > 1,
+              let statement = node.statements.last,
+              let breakStatement = statement.item.as(BreakStmtSyntax.self),
+              breakStatement.label == nil,
+              !node.isContainedIn(regions: disabledRegions, locationConverter: locationConverter) else {
+            return super.visit(node)
+        }
+
+        correctionPositions.append(statement.item.positionAfterSkippingLeadingTrivia)
+
+        let trivia = statement.item.leadingTrivia + statement.item.trailingTrivia
+
+        let stmts = node.statements.removingLast()
+        guard let secondLast = stmts.last else { return super.visit(node) }
+
+        return super.visit(
+            node.with(\.statements, stmts)
+                .with(\.statements.trailingTrivia, secondLast.item.trailingTrivia + trivia)
+                .trimmed { !$0.isLineComment }.formatted().as(SwitchCaseSyntax.self)!
```

Comment:
What about other comment types like `.blockComment`, `.docBlockComment` and `.docLineComment`?

Reply:
Fixed in 9cb04aa